packages/domains-table/src/domains-table/domains-table-ssl-cell.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon } from '@wordpress/components';
+import { lock } from '@wordpress/icons';
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+
+interface DomainsTableSSLCellProps {
+	sslStatus: 'active' | 'pending' | 'disabled' | null;
```

Comment:
```suggestion
	sslStatus: DomainData[ 'ssl_status' ];
```
Minor, but it would make things easier to reference the canonical type definition directly.

Reply:
Looks like we still need to handle this change